packages/lesswrong/server/apolloServer.ts

```diff
@@ -374,6 +377,7 @@ export function startWebserver() {
         + externalStylesPreload
         + instanceSettingsHeader
         + faviconHeader
+        + publicSettingsHeader // Must come before clientScript
```

Comment:
This is important, I've moved this to be sent as part of the prefetch section so that the settings can start being used immediately